ethers-core/src/types/trace/geth/test_data/call_tracer/default.rs

```diff
@@ -0,0 +1,21 @@
+r#"{
+    "calls": [
+        {
+            "from": "0x3b873a919aa0512d5a0f09e6dcceaa4a6727fafe",
+            "gas": "0x6d05",
+            "gasUsed": "0x0",
+            "input": "0x",
```

Comment:
I'd prefer them to be the json objects and not json strings embedded in rust file.

Reply:
get it